tests/lib/bin/mountinfo-tool

```diff
@@ -453,7 +453,13 @@ def renumber_mount_option(opt, seen):
         try:
             return seen[key]
         except KeyError:
-            n = len(seen)
+            n = len(
+                {
+                    opt_value
+                    for opt_key, opt_value in seen.keys()
+                    if opt_key == mount_opt_key
```

Comment:
Why not just a helper dict (apart from `seen`) that maps mount_opt_key to a counter and increases it on every alloc? That would avoid iterating over the dict on every alloc and would make the intent more clear.

Reply:
Because I used the same trick for the block device and because it's not a critical for performance. I'm happy to revisit after the patch set lands and we focus on reviewing the test this enables.

Reply:
Allright.